src/rupali/assignment17/ArrEvenOdd.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Program 1:  From given numbers, count the odd and even numbers.
+input : 10,23,26,27,88,87,65 
+output : even -> 3
+      odd ->  array.length - evenCount
+	  
+ */
+package rupali.assignment17;
+import java.util.Scanner;
+
+public class ArrEvenOdd {
+	
+	void getEvenOddcount() {
+		int count=0;
+		Scanner s=new Scanner(System.in);
```

Comment:
We should not ask, user to give inputs in service method,
Ideally we take input from main method then pass those value to methods while calling.